### PR TITLE
[haskell/pl-pl] Translate Haskell docs from English to Polish

### PR DESCRIPTION
**Pull request by remigiusz-suwalski**
- [x] Pull request title is prepended with `[language/lang-code]`
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [ ] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Yes, I have double-checked quotes and field names!

### CODE REVIEW

File: pl-pl/haskell-pl.html.markdown [hunk 1]
```diff
@@ -0,0 +1,444 @@
+---
+language: Haskell
+lang: pl-pl
+contributors:
+    - ["Remigiusz Suwalski", "https://github.com/remigiusz-suwalski"]
+---
+
+Haskell został zaprojektowany jako praktyczy, czysto funkcyjny język
```
Comment (Hatsuney) [inline pl-pl/haskell-pl.html.markdown]: praktyczny

**Comment by remigiusz-suwalski** [comment]
@Hatsuney the missing letter has been added.

**Comment by remigiusz-suwalski** [comment]
@Hatsuney is there something else that I can do to improve the quality of code?

**Comment by adambard** [comment]
Przepraszam, sometimes we get distracted. Looks good to me, let's let it loose and see if anyone else complains :)

**Pull request merged**